repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/tests/test_redhatsignedrpmscanner.py

```diff
@@ -16,11 +22,6 @@ class MockModel(Model):
     int_field = fields.Integer(default=42)
 
 
-def test_actor_execution(current_actor_context):
```

Comment:
Removing it as the actor should consume InstalledRPM first before execution. This is actually weird, that previously that wasn't an issue

Reply:
the test is actually valid. It tests hypothetical scenario how the actor is supposed to work in case the msg from previous actor is not produced. Just the name of the test could be different like: `test_no_installedrpms`. I know this situation should not happen, but it tells that author thought about that and decided in such a case actor should not fail.

Reply:
Fixed, thx for the explanation @pirat89